src/spasht/agent/Makefile

```diff
@@ -1,5 +1,4 @@
-# Copyright 2019
-# Author: Vivek Kumar<vvksindia@gmail.com>
+# Copyright Siemens AG 2014
```

Comment:
Please update the copyright

Reply:
Updated in new commit of feat/spasht_5.0